weave

```diff
@@ -1197,6 +1202,9 @@ launch() {
                 IPRANGE="${1#*=}"
                 IPRANGE_SPECIFIED=1
                 ;;
+            --hairpin-mode=*)
+                HAIRPIN_MODE="${1#*=}"
```

Comment:
I don't think we should bother with this; users can set the environment variable if they really want to.
(This comment is in the context that this setting only gets used in the CNI code path)